addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -201,52 +201,59 @@ def delete_bill(self, bill_id):
 
     ##create/get/delete_bills payment
     @api.model
-    def create_bill_payment(self, bill_ids, journal_id, payment_date, payment_method_line_id):
-        # Ensure the bills exist and are valid
-        AccountMove = self.env['account.move']
-        bills = AccountMove.browse(bill_ids)
-        if not bills.exists():
-            raise ValueError("No bills found with the provided IDs")
-
-        # Calculate total amount to be paid
-        total_amount = sum(bill.amount_residual for bill in bills)
-
-        # Create a context dictionary mimicking what the wizard would receive
-        payment_context = {
+    def create_bill_payment(self, bill_id, target_journal_id, payment_date, payment_method_line):
+        # Verify the specified bill exists
+        BillModel = self.env['account.move']
+        target_bill = BillModel.browse(bill_id)
+        if not target_bill.exists():
+            raise ValueError(f"Bill with ID {bill_id} was not found.")
+
+        # Determine the outstanding amount to be paid on the bill
+        payment_amount = target_bill.amount_residual
+
+        # Set up the context as expected by the payment register wizard for processing a single bill
+        wizard_context = {
             'active_model': 'account.move',
-            'active_id': bill_ids[0],
-            'active_ids': bill_ids,
+            'active_id': bill_id,
+            # 'active_ids': [single_bill_id],
```

Comment:
why did we decide to switch to single `bill_id` versus multiple `bill_ids` we used in previous version of the code?